src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -452,6 +455,17 @@ const Row = ({
 }: RowPropTypes) => {
   const [isExpanded, setIsExpanded] = useState(false);
   const handleToggle = () => setIsExpanded(!isExpanded);
+  const dispatch = useDispatch();
+
+  const handleClick = ({
+    blueprintId,
+  }: {
+    blueprintId: string | undefined | null;
+  }) => {
```

Comment:
We already have a type for blueprintId, can we use that?

```suggestion
  const handleClick = ({
    blueprintId,
  }: {
    blueprintId: BlueprintItem['id'];
  }) => {
```